resources/osl_php_yourls.rb

```diff
@@ -0,0 +1,59 @@
+resource_name :osl_php_yourls
+provides :osl_php_yourls
+unified_mode true
+
+default_action :install
+
+property :version, String
+property :fqdn, String, name_property: true
+property :db_username, String
+property :db_password, String, sensitive: true
+property :db_name, String
+property :db_host, String
+property :db_prefix, String
+property :domain, String
+property :language, String, default: ''
+property :unique_urls, Boolean, default: true
+property :private, Boolean, default: true
```

Comment:
I don't remember why, but this is the preferred method for this:
```suggestion
property :unique_urls, [true, false], default: true
property :private, [true, false], default: true
```